src/lib/integration_test_local_engine/docker_node_config.ml

```diff
@@ -81,10 +81,19 @@ module Base_node_config = struct
 
   let container_libp2p_key_path = container_keys_path ^ "/libp2p_key"
 
-  let entrypoint_script =
+  let magic_config_file commit_id =
+    Printf.sprintf "/var/lib/coda/config_%s.json" commit_id
+
+  let entrypoint_script commit_id =
     ( "entrypoint.sh"
     , {|#!/bin/bash
   # This file is auto-generated by the local integration test framework.
+  #
```

Comment:
I wonder, could we just provide more overrides in integration test code instead of crude removal. P.S. in a med-term we want to get rid of this config discovery in a docker image.